spec/features/archiving_tags_for_mainstream_browse_pages_spec.rb

```diff
@@ -0,0 +1,147 @@
+require 'rails_helper'
+require 'gds_api/test_helpers/content_store'
+
+RSpec.feature "Archiving tags for mainstream browse pages" do
+  include PublishingApiHelpers
+  include CommonFeatureSteps
+  include GdsApi::TestHelpers::ContentStore
+
+  before do
+    # Stub rummager so that topics do not have links.
```

Comment:
# Stub rummager so that topics do not have links. 

topics => Mainstream Browse Pages ?